LayoutTests/imported/w3c/web-platform-tests/resources/testdriver-vendor.js

```diff
@@ -285,9 +280,7 @@ window.test_driver_internal.action_sequence = function(sources)
             }
         });
     }
-
     logDebug(() => JSON.stringify(pointerSource));
-
     if (pointerType === "touch")
         return dispatchTouchActions(pointerSource.actions);
```

Comment:
should be `await` then `else if` to preserve old behavior, given both methods return a Promise.